lib/browser/rpc-server.js

```diff
@@ -236,9 +243,34 @@ const unwrapArgs = function (sender, contextId, args) {
   return args.map(metaToValue)
 }
 
+const callbackify = function (func) {
```

Comment:
We can probably use node's [`util.callbackify()`](https://nodejs.org/dist/latest-v10.x/docs/api/util.html#util_util_callbackify_original) here